src/Jaeger/Span.cs

```diff
@@ -118,7 +118,7 @@ private void FinishInternal(DateTime finishTimestampUtc)
             {
                 if (FinishTimestampUtc != null)
                 {
-                    Tracer.Logger.LogWarning("Span has already been finished; will not be reported again.");
+                    Tracer.Logger.LogWarning("Span has already been finished; will not be reported again. Operation: {0} Trace Id: {1} Span Id: {2}", OperationName, Context.TraceId, Context.SpanId);
```

Comment:
Please don‘t use {0}, ... but speaking names. This is not string.Format syntax but structured logging of ILogger. The logging framework will create key-value-pairs for those fields in addition to a stringified version. So just use {operationName} and so on.